.gitmodules

```
@@ -1,6 +1,6 @@
 [submodule "third_party/cccl"]
 	path = third_party/cccl
-	url = https://github.com/NVIDIA/cccl.git
+	url = https://github.com/leofang/cccl.git
```

I've made a fork under CuPy org and mirrored your branch at https://github.com/cupy/cccl/tree/fix_cupy_8203!

```suggestion
	url = https://github.com/cupy/cccl.git
```

---

Great, thanks @kmaehashi! Could you grant me (and @miscco perhaps) write/developer access to the fork, please? I think in the near future there will be some CCCL-related work needed, so it's easier for us if we can push to the fork.